src/sentry/tasks/auto_remove_inbox.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import absolute_import
+
+from sentry.db.deletion import BulkDeleteQuery
+from sentry.models import GroupInbox
+from sentry.tasks.base import instrumented_task
+
+
+@instrumented_task(name="sentry.tasks.auto_remove_inbox", time_limit=65, soft_time_limit=60)
```

Comment:
I'm not sure if this is going to reliably execute in under 60 seconds. It might be worth bumping this up since it's known this is going to potentially take a while.

Reply:
This should also be run in it's own queue most likely. @JTCunning can you help guide how we do that these days? I forget.

Reply:
Since this is just a once-ran task, default is perfectly harmless. If we wanted to put it in a queue with more of its peers, `cleanup` would be more appropriate.

Reply:
Once run? It runs every 15 minutes.